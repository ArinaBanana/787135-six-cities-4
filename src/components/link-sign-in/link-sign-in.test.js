import React from "react";
import renderer from "react-test-renderer";
import LinkSignIn from "./link-sign-in.jsx";

it(`Should render Link sign in`, () => {
  const tree = renderer.create(<LinkSignIn />).toJSON();

  expect(tree).toMatchSnapshot();
});
