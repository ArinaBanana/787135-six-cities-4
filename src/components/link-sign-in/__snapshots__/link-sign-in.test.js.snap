// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Link sign in 1`] = `
<a
  className="header__nav-link header__nav-link--profile"
  href="/login"
  onClick={[Function]}
>
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  />
  <span
    className="header__login"
  >
    Sign in
  </span>
</a>
`;
